src/test/java/org/killbill/billing/plugin/analytics/http/TestReportsServlet.java

```diff
@@ -100,7 +100,7 @@ public void testSimpleSerializationWithNULL() throws Exception {
         values.add(Arrays.asList("2013-01-01", " ", 7.0));
         values.add(Arrays.asList("2013-01-01", " ", null));
         values.add(Arrays.asList("2013-01-01", " ", ""));
-        values.add(Arrays.asList("2013-01-01", "something", null));
+        values.add(Arrays.asList("2013-01-01", "something’s", null));
```

Comment:
Could we use an accent instead, i.e something like `é`, `ô`, ...

Reply:
Added some accent characters and Japanese word as well for testing.